src/demo-app/a11y/icon/icon-a11y.html

```diff
@@ -0,0 +1,14 @@
+<section>
+  <h2>Home icon (Ligature from Material Icons font)</h2>
```

Comment:
I think we should have examples for each of the accessibility use cases listed here: https://material.angular.io/components/icon/overview